charts/fiberplane-daemon/templates/deployment.yaml

```diff
@@ -33,20 +33,47 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env: 
+            - name: TOKEN
+              value: {{ .Values.fiberplane.token | default ""}}
+            - name: DATA_SOURCES_PATH
+              value: {{ .Values.fiberplane.data_sources_path | default "/app/config/data_sources.yaml" }}
+            - name: RUST_LOG
+              value: {{ .Values.fiberplane.log_level | default "proxy=info" }}
+          volumeMounts:
+            - name: data-sources
+              mountPath: {{ .Values.fiberplane.data_sources_path | default "/app/config/data_sources.yaml" }}
+              subPath: data_sources.yaml
+          {{- if .Values.ports }}
           ports:
             - name: http
               containerPort: {{ .Values.service.port }}
               protocol: TCP
-          livenessProbe:
+          {{- end }}
+          #TODO: add liveness and readiness probes
+          {{- if .Values.livenessProbe }}
+          livenessProbe: 
             httpGet:
               path: /
```

Comment:
Will these endpoints return a 200 response? Otherwise we should add a dedicated endpoint for this to fpd

Reply:
don't think they're affecting the status but I can remove the for now. When we add the `/` endpoint to `fpd` I'll readd them